lib/UnicodeTools.gi

```diff
@@ -1018,7 +1018,7 @@ end);
 UNICODE_RECODE.f := function()
   local nam, i;
   for i in Concatenation([1..11],[13..15]) do
-    nam := Concatenation("ISO-8859-", String(i));
+    nam := MakeImmutable(Concatenation("ISO-8859-", String(i)));
```

Comment:
This string is a local variable and only used to specify the name of a record field. As far as I can see, it make no difference if it is mutable or immutable.